projects/data-encoding/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Apply diff for lib/fuzz/Cargo.toml to include v3-preview feature for fuzzing
+git apply $SRC/patch.diff
+
+# Build the fuzzers and project source code
+cd lib
+cargo fuzz build -O
```

Comment:
What's the rationale to disable debug assertions and overflow checks? Without `-O` we still get a `--release` build, which is where most of the performance will come from. Debug assertions and overflow checks have a pretty high signal to overhead ratio in my opinion.

Here's the documentation of `cargo fuzz build` says:

> By default fuzz targets are built with optimizations equivalent to `cargo build --release`, but with debug assertions and overflow checks enabled. Address Sanitizer is also enabled by default.

Reply:
Gotcha, no specific reason as such and I agree with your points.

Reply:
Switched to no `-O`